### PR TITLE
Fixes #16898: Hide remote run button for Windows agent

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/16898

### CODE REVIEW

File: webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/NodeApi.scala [hunk 1]
```diff
@@ -783,10 +791,16 @@ class NodeApiService8 (
...
+      val okNodes = nodes.toList.flatMap {
+        case (_, node) if(node.agentsName.exists(a => a.agentType == AgentType.CfeEnterprise || a.agentType == AgentType.CfeCommunity)) =>
+          Some(node)
+        case _ => None
+      }
 
       val res =
       for {
-        node <- nodes.values.toList
+        node <- okNodes
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/NodeApi.scala:803]: I would have kept the whole list here but instead  in the result for non compatible nodes i would have changed the result ro "Skipped, agent is not compatible with remote-run" or just "skipped"

Comment (fanf) [reply webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/NodeApi.scala:803]: good idea

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**